foundationdb/assets/dashboards/foundationdb_transactions_and_queues.json

```diff
@@ -0,0 +1 @@
uery1"}],"style":{"palette":"dog_classic","line_type":"solid","line_width":"normal"},"display_type":"line"}],"yaxis":{"include_zero":true,"scale":"linear","label":"","min":"auto","max":"auto"},"markers":[]},"layout":{"x":4,"y":0,"width":4,"height":2}},{"id":6259859862527082,"definition":{"title":"Completed Read operations","title_size":"16","title_align":"left","show_legend":true,"legend_layout":"auto","legend_columns":["avg","min","max","value","sum"],"type":"timeseries","requests":[{"formulas":[{"alias":"Reads / second","formula":"query1"}],"response_format":"timeseries","queries":[{"query":"avg:foundationdb.workload.operations.reads.hz{$service}","data_source":"metrics","name":"query1"}],"style":{"palette":"dog_classic","line_type":"solid","line_width":"normal"},"display_type":"line"}],"yaxis":{"include_zero":true,"scale":"linear","label":"","min":"auto","max":"auto"},"markers":[]},"layout":{"x":8,"y":0,"width":4,"height":2}},{"id":3689974919495706,"definition":{"title":"Write Operations Completed","title_size":"16","title_align":"left","show_legend":true,"legend_layout":"auto","legend_columns":["avg","min","max","value","sum"],"type":"timeseries","requests":[{"formulas":[{"alias":"Writes / second","formula":"query1"}],"response_format":"timeseries","queries":[{"query":"avg:foundationdb.workload.operations.writes.hz{$service}","data_source":"metrics","name":"query1"}],"style":{"palette":"dog_classic","line_type":"solid","line_width":"normal"},"display_type":"line"}],"yaxis":{"include_zero":true,"scale":"linear","label":"","min":"auto","max":"auto"},"markers":[]},"layout":{"x":0,"y":2,"width":4,"height":2}},{"id":1246848430284440,"definition":{"title":"Proxy memory limit errors","title_size":"16","title_align":"left","show_legend":true,"legend_layout":"auto","legend_columns":["avg","min","max","value","sum"],"type":"timeseries","requests":[{"formulas":[{"alias":"Errors / second","formula":"query1"}],"response_format":"timeseries","queries":[{"query":"avg:foundationdb.workload.operations.memory_errors.hz{$service}","data_source":"metrics","name":"query1"}],"style":{"palette":"dog_classic","line_type":"solid","line_width":"normal"},"display_type":"line"}],"yaxis":{"include_zero":true,"scale":"linear","label":"","min":"auto","max":"auto"},"markers":[]},"layout":{"x":4,"y":2,"width":4,"height":2}},{"id":8304484655265424,"definition":{"type":"note","content":"Memory errors are requests rejected by proxies because they are reached their memory limit, which indicates they already have too many requests.","background_color":"white","font_size":"14","text_align":"left","vertical_align":"top","show_tick":false,"tick_pos":"50%","tick_edge":"left","has_padding":true},"layout":{"x":8,"y":2,"width":4,"height":2}}]},"layout":{"x":0,"y":8,"width":12,"height":5}}],"template_variables":[{"name":"service","default":"*","prefix":"service","available_values":[]}],"layout_type":"ordered","is_read_only":false,"notify_list":[],"reflow_type":"fixed","id":"t4e-pxz-2xz"}
```

Comment:
This blog post link is broken: http://datadoghq.com/blog/monitor-foundationdb-with-datadog. Let's remove this for now and add it back when a blog post is written.

Reply:
Sorry about that, I've synced up with product and the blog post will point to that URL when it is published.